### DISCUSSION TITLE
How can I change global TextFontSize?

### DISCUSSION THREAD

**Discussion by JerryJian** [HandyOrg/HandyControl#1214] [Q&A]
The default TextFontSize is 12 and is too small. 
How can I change it to 14?

**Comment by Yopler** [comment]
Hi 🖐,
Changing the `FontSize` property in your `Window` element should work.
```xaml
<Window 
        ...
        FontSize="100">
    <TextBlock Text="Yoooooooooooooo"/>
</Window>
```

You can read the following stackoverflow question : https://stackoverflow.com/questions/893428/wpf-global-font-size

Hope it can help 😊
